src/settings.c

```diff
@@ -160,12 +160,16 @@ void settings_init(void) {
         }
 }
 
+// NOTE: This function is probably outdated and no longer relevant
+//       Since it does not even fully display rule information we
+//       may want to remove it or change it
```

Comment:
I'm fine with removing it ... it basically just echoes the user input. The only use case could be to find typos in the configuration, but a) is this just printed at DEBUG level and b) is there already a WARNING issued, whenever an unknown key is encountered.

Reply:
The main problem is that it prints only some values, so it doesn't really help that much